lightning/common/util.go

```diff
@@ -102,11 +102,11 @@ func IsEmptyDir(name string) bool {
 // SQLWithRetry constructs a retryable transaction.
 type SQLWithRetry struct {
 	DB           *sql.DB
-	Logger       log.Logger
+	Logger       *zap.Logger
```

Comment:
we could keep using `log.Logger` here. external packages just need to write `log.Logger{Logger: logger}` to wrap a `*zap.Logger`.

Reply:
this change is to enable TiDB's implementation of glue get rid of lightning's logger, not sure if there's a good way